sbt/src/sbt-test/source-dependencies/macro/build.sbt

```diff
@@ -0,0 +1,21 @@
+val defaultSettings = Seq(
+  libraryDependencies <+= scalaVersion("org.scala-lang" % "scala-reflect" % _ )//,
+  //incOptions := incOptions.value.withNameHashing(true)
+)
+
+lazy val root = (project in file(".")).
+  aggregate(macroProvider, macroClient).
+  settings(
+    defaultSettings
+  )
+
+lazy val macroProvider = (project in file("macro-provider")).
```

Comment:
Shouldn't you use "lazy val \`macro-provider\` = ..." to retain the same project id as before? (id = "macro-provider"). Same issue in a bunch of other places.